bin/uaparser

```diff
@@ -2,6 +2,12 @@
 <?php
 namespace UAParser\Command;
 
+// Ensure symfony packages installed
+if (!class_exists(Symfony\Component\Console\Application::class)) {
```

Comment:
this will only work after autoloading was included, I guess.

Reply:
Na, php will convert that to a string regardless of autoloading. Allows for cleaner refactoring and usage finders though.

Reply:
I mean it will never find this symfony classes as the composer autoloading is required a few lines below

Reply:
`php -a`
```
php > var_dump(Foo\Bar::class);
string(7) "Foo\Bar"
```
The autoloader isn't required, PHP just interpolates that syntax into a string. magic 😄

Reply:
But it cannot find the class when composer is not required

Reply:
I promise you that isn't the case. PHP doesn't invoke the autoloader when doing `::class` on the end of a class. Exactly why running that snipped in an interactive shell with no autoloader registered doesn't cause a `class not found` error

Reply:
Nothing better than a small experiment to show and tell :smiley: 

```
<?php
namespace UAParser\Command;

echo Symfony\Component\Console\Application::class .
    (class_exists(Symfony\Component\Console\Application::class) ? ' exists' : ' does not exist'), PHP_EOL;

require_once(__DIR__.'/../vendor/autoload.php');

echo Symfony\Component\Console\Application::class .
    (class_exists(Symfony\Component\Console\Application::class) ? ' exists' : ' does not exist'), PHP_EOL;
```

The output may surprise you, it is:

```
UAParser\Command\Symfony\Component\Console\Application does not exist
UAParser\Command\Symfony\Component\Console\Application does not exist
```

The problem is that the `Symfony\Component\Console\Application` class needs to be prefixed with a \ to indicate it lives in the root namespace and is not a part of this namespace (the `use` statement later on does not change this). With that fixed, let's try again:

```php
<?php
namespace UAParser\Command;

echo \Symfony\Component\Console\Application::class .
    (class_exists(\Symfony\Component\Console\Application::class) ? ' exists' : ' does not exist'), PHP_EOL;

require_once(__DIR__.'/../vendor/autoload.php');

echo \Symfony\Component\Console\Application::class .
    (class_exists(\Symfony\Component\Console\Application::class) ? ' exists' : ' does not exist'), PHP_EOL;
```

and the output is now:

```
Symfony\Component\Console\Application does not exist
Symfony\Component\Console\Application exists
```

So basically:

1. The check needs to be performed _after_ the autoloader has been loaded
2. The check needs to be for `\Symfony\Component\Console\Application::class` instead of `Symfony\Component\Console\Application::class`